Modules/_csv.c

```diff
@@ -1410,8 +1417,18 @@ csv_writer(PyObject *module, PyObject *args, PyObject *keyword_args)
 /*
  * DIALECT REGISTRY
  */
+
+/*[clinic input]
+_csv.list_dialects
+
+Return a list of all know dialect names.
```

Comment:
```suggestion
Return a list of all known dialect names.
```

Reply:
Thanks, although I'll note that the error was extant (I can revert the typo fix if someone feels that's a problem).